### PR TITLE
Remove netty dependency from api cr1update

### PR DESCRIPTION
**Pull request by artem-v**

### CODE REVIEW

**Comment by ronenhamias** [comment]
tests are failing as result

**Review by ronenhamias — DISMISSED** [review]
tests are failing

**Review by segabriel — APPROVED** [review]
:+1:

File: services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGateway.java [hunk 1]
```diff
@@ -24,7 +26,9 @@ public HttpGateway(GatewayOptions options) {
   public Mono<Gateway> start() {
     return Mono.defer(
         () -> {
-          HttpGatewayAcceptor acceptor = new HttpGatewayAcceptor(options.call(), gatewayMetrics);
+          ServiceCall serviceCall =
+              options.call().requestReleaser(ReferenceCountUtil::safestRelease);
```
Comment (ronenhamias) [inline services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGateway.java]: lets rename to release

**Review by segabriel — APPROVED** [review]

**Pull request merged**